### PR TITLE
[FIF-8] Add initial projects

### PR DESCRIPTION
**Pull request by kurtjohnson-HS**
This commit creates two new projects for FixItFriday - a netcoreapp3.1 targeted WebApi project (FixItFriday.Api) and a React app (fixitfriday.ui). 

NOTE: fixitfriday.ui is intentional because the [create-react-app requires lowercase project names](https://github.com/facebook/create-react-app/issues/2165#issuecomment-301627339).

### CODE REVIEW

File: fixitfriday.ui/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "fixitfriday.ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
```
Comment (stephenfuqua) [inline fixitfriday.ui/package.json:8]: Should the testing libraries be dev-dependencies?

File: fixitfriday.ui/src/App.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (stephenfuqua) [inline fixitfriday.ui/src/App.js:1]: Just curious - did you guys discuss the possibility of using Typescript? Not sure how valuable that is in a pure front-end application where you have a strong API handling most logic. So truly just curious here and not trying to suggest a direction.

Comment (kurtjohnson-HS) [reply fixitfriday.ui/src/App.js:1]: We did and are in favor of using TypeScript. I created the initial commit as close to the template so any of the infrastructure decisions can be additive.

File: fixitfriday.ui/package.json (hunk 1 above)
Comment (kurtjohnson-HS) [reply fixitfriday.ui/package.json:8]: Jest comes by default in the create-react-app. The team felt that having UI testing had a low value with the read-only requirement. I didn't specifically remove this dependency yet. If the UI testing removal is agreed, we can uninstall this.

Comment (stephenfuqua) [reply fixitfriday.ui/package.json:8]: OK

File: fixitfriday.ui/src/services/graphqlClient.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import ApolloClient from "apollo-boost";
+
+const client = new ApolloClient({
+  uri: 'https://localhost:port', //URL of the GraphQL server
```
Comment (kurtjohnson-HS) [inline fixitfriday.ui/src/services/graphqlClient.js:4]: Consider changing port to 5001 to match the [FixItFriday.API\properties\localSettings.json](https://github.com/Ed-Fi-Alliance-OSS/Fix-It-Friday/pull/3/files#diff-361138310dd8a23dfec48962aef663f5R24)

File: fixitfriday.ui/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (kurtjohnson-HS) [inline fixitfriday.ui/public/manifest.json:3]: Consider updating this to Fix It Friday for now until we direction for a new setting.

File: fixitfriday.ui/src/services/graphqlClient.js (hunk 3 above)
Comment (lalrg) [reply fixitfriday.ui/src/services/graphqlClient.js]: Done

File: fixitfriday.ui/public/manifest.json (hunk 4 above)
Comment (lalrg) [reply fixitfriday.ui/public/manifest.json]: Since this is not intended to be a PWA, I removed the manifest

**Pull request merged**